internal/view/issues.go

```diff
@@ -0,0 +1,113 @@
+package view
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"strings"
+)
+
+type Issue struct {
+	Severity     SeverityLevel
+	Description  string
+	IntroducedBy []string
+	SnykRef      string
+}
+
+type issue struct {
+	Severity     string
+	Description  string
+	IntroducedBy []string
+	SnykRef      string
+}
+
+type issues struct {
+	title  string
+	issues []issue
+
+	str string
+}
+
+// GenerateIssues constructs a list of issues and severities and generates it's
+// string representation intended for human readable output.
+//
+// Function returns an error if generation of string representation fails.
+func GenerateIssues(severities ...Issue) (*issues, error) {
+	result := issues{
+		title:  sectionStyle.Render("Open issues:"),
+		issues: make([]issue, len(severities)),
+	}
+
+	for i := range severities {
```

Comment:
in the snapshot, issues are out of order. imho this would be a good place to apply the sorting, so the consumer of this package doesn't have to and we get consistent behaviour. wdyt?

Reply:
I decided to place the sorting within the render since it's still in the presentation layer while each component could be a naive layer.

Another reason is because then it's easier to expose to the user what and when it's dangerous to pass a slice, since it's mutating existing data.